api/main_endpoints/models/Event.js

```diff
@@ -30,9 +30,10 @@ const EventSchema = new Schema(
     eventCategory: {
       type: String
     },
+
     imageURL: {
       type: String,
-      default: DEFAULT_PHOTO_URL
+      default: 'https://www.freeiconspng.com/uploads/no-image-icon-11.PNG'
```

Comment:
can you keep the `DEFAULT_PHOTO_URL` here?

Reply:
Yup I can keep it